scripts/lc-builds/blueos_nvcc_clang.sh

```diff
@@ -16,12 +31,13 @@ module load cmake/3.14.5
 
 cmake \
   -DCMAKE_BUILD_TYPE=Release \
-  -DCMAKE_CXX_COMPILER=/usr/tce/packages/clang/clang-upstream-2019.08.15/bin/clang++ \
+  -DCMAKE_CXX_COMPILER=/usr/tce/packages/clang/clang-${COMP_CLANG_VER}/bin/clang++ \
+  -DBLT_CXX_STD=c++11 \
```

Comment:
This is BLT_CXX_STD is a long awaited good addition. In one of my branches I added this in the top level CMakeLists.txt, not sure if you want to move it there?

Reply:
I don't think we want to force it since BLT doesn't and users should be aware of it.